src/lib/db.ts

```diff
@@ -616,4 +671,37 @@ export class DB {
         return workspace;
       });
   }
+
+  private parseDocumentContent(document: DBDocument): DocumentContentModelType|null {
+    if (!document.content) {
+      return null;
+    }
+
+    const storedContent: DocumentContentModelType = JSON.parse(document.content);
+    return DocumentContentModel.create({
```

Comment:
I think you can just pass the raw parsed JSON to `DocumentContentModel.create()`

Reply:
Wow yes you'd think I'd have realized that given that I was able to declare this as a `DocumentContentModelType` right away 😅Good call!